build.gradle.kts

```diff
@@ -37,6 +37,8 @@ dependencies {
     implementation(kotlin("gradle-plugin"))
     modCompileOnly("cc.polyfrost:oneconfig-${mcData.versionStr}-${mcData.loader.name}:0.2.0-alpha+")
     implementation(shade("cc.polyfrost:oneconfig-wrapper-launchwrapper:1.0.0-beta+")!!)
+    minecraft("com.mojang:minecraft:1.8.9")
```

Comment:
What this?

Reply:
idk thats already in the mod anyway (just took like 7 attempts to spell annyway)